### ISSUE TITLE
try out aligncheck on our structs

### ISSUE THREAD

**Issue by slingamn** [oragono/oragono#471]
Probably not even worth doing, but might be interesting to see how much space we can save on `Client` by reordering the struct fields:

https://gitlab.com/opennota/check

**Comment by slingamn** [comment]
````
github.com/oragono/oragono/irc: /home/shivaram/go/src/github.com/oragono/oragono/irc/channel.go:24:6: struct Channel could have size 392 (currently 400)
github.com/oragono/oragono/irc: /home/shivaram/go/src/github.com/oragono/oragono/irc/client.go:48:6: struct Client could have size 688 (currently 736)
github.com/oragono/oragono/irc: /home/shivaram/go/src/github.com/oragono/oragono/irc/config.go:178:6: struct NickReservationConfig could have size 48 (currently 56)
github.com/oragono/oragono/irc: /home/shivaram/go/src/github.com/oragono/oragono/irc/config.go:230:6: struct STSConfig could have size 40 (currently 48)
github.com/oragono/oragono/irc: /home/shivaram/go/src/github.com/oragono/oragono/irc/config.go:56:6: struct AccountConfig could have size 416 (currently 432)
github.com/oragono/oragono/irc: /home/shivaram/go/src/github.com/oragono/oragono/irc/help.go:28:6: struct HelpEntry could have size 40 (currently 48)
github.com/oragono/oragono/irc: /home/shivaram/go/src/github.com/oragono/oragono/irc/server.go:957:6: struct elistMatcher could have size 24 (currently 32)
github.com/oragono/oragono/irc: /home/shivaram/go/src/github.com/oragono/oragono/irc/socket.go:30:6: struct Socket could have size 112 (currently 120)
````

tl;dr small potatoes compared to the potential gains from optimizing the client's read buffer size. (Right now, we eagerly expand the buffer to the maximum possible size of the client line --- around 6 KB --- even though most clients will never send a message over 1 KB.)
